PickMeUp/PickMeUp/PMPostCell.h

```diff
@@ -0,0 +1,24 @@
+//
+//  PostCellTableViewCell.h
+//  PickMeUp
+//
+//  Created by Abigail Shilts on 7/7/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "Post.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PMPostCell : UITableViewCell
+@property (weak, nonatomic) IBOutlet UIImageView *groupImg;
+@property (weak, nonatomic) IBOutlet UILabel *groupIntensity;
+@property (weak, nonatomic) IBOutlet UILabel *groupSport;
+@property (weak, nonatomic) IBOutlet UILabel *groupLocation;
+@property (weak, nonatomic) IBOutlet UILabel *groupTime;
+@property Post *post;
```

Comment:
Don't forget (nonatomic, strong) in the property definition
But do you need to retain Post in your cell? if you use it to update UI components and throw it away after just create a method `updateWith:(Post*)post` and do everything there